examples/discovery-search-app/cypress/integration/collection_filter/spec.ts

```diff
@@ -34,13 +34,13 @@ describe('Collection Filter', () => {
       });
 
       it('displays a list of the collections in the project', () => {
-        cy.get('#collection-facet-select__menu')
+        cy.get('.bx--list-box__menu')
```

Comment:
Why are you switching to classes here?

Reply:
i think when i upgraded carbon accidentally it stopped passing the `id` parameter to the internal downshift components, thus this selector no longer existed. im open to other selectors, the other one being `#downshift-0-menu`

Reply:
Ahh ok. I would prefer an id if possible. but if not since we don't control it, that is ok.